HLTrigger/Configuration/python/Tools/confdbOfflineConverter.py

```diff
@@ -109,6 +119,16 @@ def __init__(self, version = 'v3', database = 'run3', url = None, verbose = Fals
         else:
             self.proxy_connect_args = ()
 
+        #this sets the host to localhost
+        if self.tunnel:
+            temp_connect = list(self.connect)
+            host_index = temp_connect.index('-h')
+            temp_connect[host_index+1] = "localhost"
+            self.connect = tuple(temp_connect)
+            self.tunnel_connect_args = ('--dbport',self.tunnelPort)
+        else:
+            self.tunnel_connect_args
```

Comment:
```suggestion
            self.tunnel_connect_args = ()
```
Not sure if this works without the `= ()`.

Reply:
ah thanks!